quickstart/gcp/service-account.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Setup Service Account
+---
+
+[Google Cloud Platform Credentials]: https://console.cloud.google.com/apis/credentials
+
+You will need to provide the Google Cloud Platform Provider with your Google service account private key. You can
```

Comment:
Perhaps an intro paragraph about the scenario and value for using a Service Account - to be able to run Pulumi deployments with a specific set of scoped credentials not associated with an individual user - whether in CI or for more control over the account role used locally for development.

Reply:
Added this:

>Using a Google service account allows you to use Pulumi in a non-interactive setting (for example CI/CD systems, where a person can not complete the normal `gcloud auth application-default login` flow). A service account can also be used when developing locally to ensure a specific set of scoped credentials not tied to a user account are used. This can be useful even when developing locally to give you more control over the account role used for deployment.

To the start of the page.